### DISCUSSION TITLE
Tuya Smart Valve TS011F being detected as smart plug after last update to 1.31.0

### DISCUSSION THREAD

**Discussion by jorgeveca** [Koenkk/zigbee2mqtt#17882] [Devices]
Hello All,

Tuya Smart Valve TS011F being detected as smart plug after last update to 1.31.0.

Before this update it was working as expected. Now, the image change to smart plug and I am seeing errors in the logs about getting state of the device.

Jun 02 11:00:46 Zigbee2MQTTPi4-PB npm[243375]: Zigbee2MQTT:error 2023-06-02 11:00:46: Failed to configure 'Llave Tanque del Agua', attempt 4 (Error: Bind 0xa4c1389a4459e49a/1 haElectricalMeasurement from '0x00212effff075e46/1' failed (Error: status: 130)
Jun 02 11:00:46 Zigbee2MQTTPi4-PB npm[243375]:     at DeconzAdapter.bind (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/adapter/deconz/adapter/deconzAdapter.ts:766:19)
Jun 02 11:00:46 Zigbee2MQTTPi4-PB npm[243375]:     at Endpoint.bind (/opt/zigbee2mqtt/node_modules/zigbee-herdsman/src/controller/model/endpoint.ts:645:13)
Jun 02 11:00:46 Zigbee2MQTTPi4-PB npm[243375]:     at Object.bind (/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/src/lib/reporting.js:40:9)
Jun 02 11:00:46 Zigbee2MQTTPi4-PB npm[243375]:     at Object.configure (/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/src/devices/tuya.js:3170:13)
Jun 02 11:00:46 Zigbee2MQTTPi4-PB npm[243375]:     at Configure.configure (/opt/zigbee2mqtt/lib/extension/configure.ts:117:13)
Jun 02 11:00:46 Zigbee2MQTTPi4-PB npm[243375]:     at Configure.onMQTTMessage (/opt/zigbee2mqtt/lib/extension/configure.ts:55:21))
Jun 02 11:03:08 Zigbee2MQTTPi4-PB npm[243375]: Zigbee2MQTT:error 2023-06-02 11:03:08: Publish 'get' 'state' to 'Llave Tanque del Agua' failed: 'Error: Read 0xa4c1389a4459e49a/1 genOnOff(["onOff"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Status 'WRITE_ONLY')'

Does anyone know how to fix it? Thanks.

**Comment by Koenkk** [comment]
Please fill in https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=wrong+device&projects=&template=wrong_device.yaml&title=%5BWrong+device%5D%3A+

**Comment by lean31415** [comment]
Same Issue here...

**Reply by andrei-lazarov** [reply]
Please open a new issue and provide some more details
